specification/schemaregistry/SchemaRegistry/tspconfig.yaml

```
@@ -0,0 +1,39 @@
+parameters:  
+  "service-dir":
+    default: "sdk/schemaregistry"
+  "dependencies":
+    "additionalDirectories": []
+    default: ""
+emit:
+  - "@azure-tools/typespec-autorest"
+linter:
+  extends:
+    - "@azure-tools/typespec-azure-core/all"
+options:
+  "@azure-tools/typespec-autorest":
+    azure-resource-provider-folder: "data-plane"
+    emitter-output-dir: "{project-root}/../data-plane/SchemaRegistry/preview/2023-07-01"
+    output-file: "SchemaRegistry.json"
+    examples-directory: examples
+  "@azure-tools/typespec-python":
+    package-dir: "azure-schemaregistry"
+    package-name: "{package-dir}"
+    package-version: 1.3.0b4
+    emitter-output-dir: "{project-root}/{service-dir}/{package-name}"
+    generate-packaging-files: false
+  "@azure-tools/typespec-java":
+    package-dir: "azure-data-schemaregistry"
+    namespace: com.azure.data.schemaregistry
+    partial-update: true
+    examples-directory: examples
+  "@azure-tools/typespec-ts":
+    package-dir: "schema-registry"
+    generateMetadata: false
+    generateTest: false
+    packageDetails:
+      name: "@azure/schema-registry"
```

```suggestion
      name: "@azure-rest/schema-registry"
```
Is this the same package as https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/schemaregistry/schema-registry this one? if it's a different one, we should use a different package name.

---

this is the same package

---

The previous schema-registry js library contains a lot of manual code, and it's a high level client, while RLC is a different rest level client, if you insist to use the generated code to replace the existing library, it may contains big breaking changes, are you sure you want that?
If yes, you may specify the 
```
emitter-output-dir: "{js-sdk-folder}/sdk/{service-directory-name}/schema-registry"
```

---

We'll be adding necessary API to the customizations layer, so there shouldn't be any breaking changes as far as I know. Are there any in particular that you're concerned about @qiaozha?